src/main/java/com/codeus/winter/annotation/InitDestroyAnnotationBeanPostProcessor.java

```diff
@@ -0,0 +1,75 @@
+package com.codeus.winter.annotation;
+
+import com.codeus.winter.config.BeanPostProcessor;
+import com.codeus.winter.config.DestructionBeanPostProcessor;
+import com.codeus.winter.exception.BeanNotFoundException;
+import jakarta.annotation.Nullable;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * BeanPostProcessor implementation that invokes annotated init and destroy methods. Processes
+ * methods that annotated with @PostConstruct and @PreDestroy classes
+ */
+@SuppressWarnings("java:S3011")
+public class InitDestroyAnnotationBeanPostProcessor implements BeanPostProcessor,
```

Comment:
Question:
It looks like your BeanPostProcessor calls the init method, just like my BeanFactory implementation https://github.com/DimaGalchenko/winter-framework/blob/11812a85c23a49aad283cd928cbd551f457ea1e5/src/main/java/com/codeus/winter/config/DefaultBeanFactory.java#L236.
It might make sense to remove that logic from my code and leave it here. And once the bean is initialized, this processor will be applied. What do you think?